bundles/org.eclipse.equinox.launcher/src/org/eclipse/equinox/launcher/Main.java

```diff
@@ -138,8 +140,8 @@ public class Main {
 	private String exitData = null;
 
 	private String vm = null;
-	private String[] vmargs = null;
-	private String[] commands = null;
+	private List<String> vmargs = null;
+	private List<String> commands = null;
```

Comment:
Maybe during cleaning, pointless explicit initialization to the default value could be removed too…